src/App.js

```diff
@@ -10,13 +10,20 @@ class App extends Component {
     this.setState({ list: newList });
   };
 
+  addNewItem = formData => {
+    const { list } = this.state;
+    const newList = [...list, formData];
+    this.setState({ list: newList });
+  };
```

Comment:
How are you handling the id for these newly created items. I recall you're using that to move them between the different carts.